src/index.ts

```diff
@@ -182,12 +188,12 @@ async function bundleMDX(
       }),
       nodeResolve({extensions: ['.js', '.ts', '.jsx', '.tsx']}),
       inMemoryPlugin,
+      xdmESBuild(),
```

Comment:
Turns out the only time the plugin would ever be used is if someone imports a `.mdx?` file from `node_modules`. Otherwise it'll hit our own inMemory plugin. So it's unlikely to be very useful, but I'll stick it in there anyway just in case.